src/themes/dark.scss

```diff
@@ -17,6 +16,8 @@
   --wxc-modal-background: #{$wxc-gray-100};
   --wxc-modal-border: 1px solid #{$wxc-black-alpha-11};
   --wxc-modal-box-shadow: 0 1.5rem 3rem #{$wxc-black-alpha-20}, 0 0 0.0625rem #{$wxc-black-alpha-12};
+  
+  --wxc-video-background: #{$wxc-white-alpha-05};
```

Comment:
please add this to the top just after `--wxc-text-color`, it's a generic variable, not directly connected to any component

Reply:
and please add a jira link